### ISSUE TITLE
LFPS Workshop: The Homepage (and local site)

### ISSUE THREAD

**Issue by michaelschofield** [wherebyus/general-tasks#1092]
## Problem
If we agree that each product should fulfill a specific service, we don't know what the function of the homepage is. 

So, we're going to schedule an hour-long *laser-focused positioning statement workshop* / meeting to determine that. 

## Definition of done
- [x] A laser-focused positioning statement for the homepage
- [ ] A laser-focused positioning statement for other parts of the local site
- [x] A value judgment and follow-up action about _how well our homepage [etc] fulfills its function_

**Comment by ErnieAtLYD** [comment]
@michaelschofield @carogea open to the rest of the company? Kept between PlatformProduct?

**Comment by carogea** [comment]
@ErnieAtLYD @michaelschofield i'd say these workshops can be mostly product team but we can invite some stakeholders for the specific products we discuss. Feels like a good way to get everyone on the same page and involve others in "product thinking"

https://www.niemanlab.org/2018/12/a-focus-on-problems-not-platforms/ <- a smart article that gets at what we want to do here. 

For example, a similar statement the Skimm has for their daily email newsletter is "we don’t believe the Daily Skimm is an email product. The Daily Skimm solves the problem of catching you up on what’s happening in the world so you feel like a confident, informed person starting your day. Today, that problem is well suited for email — the first thing our audience does when they wake up in the morning is check their email on their phone — but tomorrow that can readily adapt into to behaviors and routines our audience moves to instead."

**Comment by michaelschofield** [comment]
Here's @carogea's notes  https://docs.google.com/document/d/1M0-ec63SObwTezgcvvkxzFPOW-FF5qIX8Vwl0yimZZA/edit

TL;DR: the goal of the homepage is to get people to join the community.  We don't think our homepages communicate that we represent a larger community well, or that we're anything more than a newsletter. 

The question it should answer: what is and how can I be a part of this? 

We think that the vehicle for joining the community can still be the newsletter, as that aligns still with total weekly opens. 

### Other takeaways
We think there are a ton of opportunities for
- humanizing us and our community (show the people behind the articles, the people in the events, put our member spotlights out in the open)
- showing off how _companies_ and _organizations_ are also part of the company (e.g., sponsors)
- showing what kind of content is _in_ the newsletter (quartz shows the newsletter in a phone on the homepage to solve this).

### Possible action items
- We think girlsnightin call to action does what our c2a does a little better, in terms of copy, showing off _people_, and stuff. A small step might be update our c2a photos and copy to show that.

**Comment by carogea** [comment]
I think subscriber endorsements i.e. tweets mentioning how much they like TNT etc could also be great if we have them
